packages/strapi-admin/admin/src/containers/NewNotification/Notification/index.js

```diff
@@ -0,0 +1,164 @@
+import React, { useEffect, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { Padded, Text, Flex } from '@buffetjs/core';
+import { useIntl } from 'react-intl';
+import { useDispatch } from 'react-redux';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Remove } from '@buffetjs/icons';
+
+import { HIDE_NEW_NOTIFICATION } from '../constants';
+import { NotificationWrapper, IconWrapper, LinkArrow, RemoveWrapper } from './styledComponents';
+
+const types = {
+  success: {
+    icon: 'check',
+    color: 'green',
+  },
+  warning: {
+    icon: 'exclamation',
+    color: 'orange',
+  },
+  info: {
+    icon: 'info',
+    color: 'blue',
+  },
+};
+
+const Notification = ({ notification }) => {
+  const { formatMessage } = useIntl();
+  const dispatch = useDispatch();
+  const { title, message, link, type, id, onClose, timeout, blockTransition } = notification;
+
+  const formattedMessage = msg => (typeof msg === 'string' ? msg : formatMessage(msg));
```

Comment:
In order to be able to use variables in a message can you make the following modification?
`formatMessage(msg, msg.values)`